pkg/kube/inject/testdata/inject/hello-tproxy.yaml.injected

```diff
@@ -144,7 +144,7 @@ spec:
           readOnlyRootFilesystem: true
           runAsGroup: 1337
           runAsNonRoot: false
-          runAsUser: 0
+          runAsUser: 1337
```

Comment:
we should not have to change this testdata as cluster-wide should be considered the default deployment model and that should behave exactly like upstream. we only need the solution that doesn't require namespace privileges for multitenant mode